src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -554,4 +512,49 @@
   <data name="DeserializeUnableToAssignNull" xml:space="preserve">
     <value>Unable to assign 'null' to the property or field of type '{0}'.</value>
   </data>
-</root>
+  <data name="SerializerConverterFactoryReturnsJsonConverterFactory" xml:space="preserve">
```

Comment:
I'm assuming you only wanted to add this message and the tooling is responsible for all the other reorderings? I would recommend reverting the tooling changes to make the diff easier to review.

Reply:
Ok, no problem.

Reply:
I'm sorry it was my bad. I reordered the messages because they looked better in the IDE, but I didn't figure out that the reordering was hard to check in the diff :)